src/core/utils/useGetCleanBranches.tsx

```diff
@@ -0,0 +1,43 @@
+import { AgencyBranch } from "../fbs/model";
```

Comment:
Please rename this file.

I suggest something like `branches.tsx`. In the current form it contains exported functions besides the `useGetCleanBranches` hook and other files use these exported functions as well.